tests/bot/bot_test.py

```diff
@@ -0,0 +1,20 @@
+"""Test Bot Class."""
+from bot.bot import Bot
+
+def test_send_dm():
+    """Test the Bot class method send_dm()."""
+    bot = Bot()
```

Comment:
These unit tests should probably be using mocks, so that we're not sending messages every time we run the unit tests.